pkgs/development/python-modules/sphinx/default.nix

```diff
@@ -106,6 +106,12 @@ buildPythonPackage rec {
     # requires imagemagick (increases build closure size), doesn't
     # test anything substantial
     "test_ext_imgconverter"
+
+    # fails with pygments 2.14
+    # TODO remove for sphinx 6
+    "test_viewcode"
+    "test_additional_targets_should_be_translated"
+    "test_additional_targets_should_not_be_translated"
```

Comment:
Does this break the actual code? I am a bit reluctant.

Reply:
Judging by https://github.com/sphinx-doc/sphinx/commit/965768bfda2a00ba6466cdb12a7a46efdce47023 it seems like minor formatting differences.